pkg/api/configsync/v1alpha1/resource_override.go

```diff
@@ -66,6 +66,13 @@ type OverrideSpec struct {
 	// support pulling remote bases from public repositories.
 	// +optional
 	EnableShellInRendering *bool `json:"enableShellInRendering,omitempty"`
+
+	// logLevels specify the container name and log level override value for the reconciler deployment container.
+	// Each entry must contain the name of the reconciler deployment container and the desired log level.
+	// +listType=map
+	// +listMapKey=containerName
+	// +optional
+	LogLevels []ContainerLogLevelOverride `json:"logLevels,omitempty"`
```

Comment:
Should we have e2e tests that cover each api version, or is it sufficient to just have test coverage for a field on one api version?

cc @janetkuo @nan-yu @karlkfi

Reply:
We have a few tests that cover both v1apha1 and v1beta1, especially the overrides fields. If we test both api versions for all fields, a lot of tests need to be added.

I'm okay that we're super cautious when adding new fields to both APIs and only test the v1beta1.